examples/basic/main.tf

```diff
@@ -55,5 +55,7 @@ module "ocean-aks-np" {
   vmsizes_filters_min_disk                 = 1
   shutdown_hours                           = { is_enabled = false,
                                                time_windows = ["Fri:15:30-Sat:13:30", "Sun:15:30-Mon:13:30"] }
+  should_roll                              = true
+  batch_size_percentage                    = 33
```

Comment:
update the value to default

Reply:
Updated to 25, as per the the value mentioned in API doc.